scripts/aws_ecs_benchmarks_cdn.sh

```diff
@@ -34,54 +34,67 @@ just async_std example cdn-broker -- -d redis://localhost:6379 \
     --private-bind-endpoint 0.0.0.0:1741 \
     --private-advertise-endpoint local_ip:1741 &
 # server2: broker
-# just async_std example cdn-broker -- -d ${keydb_address} \
-#     --public-bind-endpoint 0.0.0.0:1740 \
-#     --public-advertise-endpoint local_ip:1740 \
-#     --private-bind-endpoint 0.0.0.0:1741 \
-#     --private-advertise-endpoint local_ip:1741 &
+# make sure you're able to access the remote host from current host
+REMOTE_USER="sishan"
+REMOTE_HOST="3.135.239.251"
```

Comment:
If possible, I'd like to keep these parameterized so we aren't committing this to the repo :)